### PR TITLE
MGMT-3390 - Added lint and format - for now, lint not performed automatically

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

**Review by YuviGold — APPROVED** [review]
LGTM

**Comment by ronniel1** [comment]
Removed the `downstream-change` label since the change is in the build dockerfile.

@gamli75  no need to modify downstream, right?

**Comment by gamli75** [comment]
@ronniel1 no need to change the DS, juts remove the label

File: Makefile [hunk 1]
```diff
@@ -17,12 +17,19 @@ DOCKER_COMPOSE=docker-compose -f ./subsystem/docker-compose.yml
 
 all: build
 
+lint: format
+	golangci-lint run -v
+
+format:
+	goimports -w -l src/ subsystem/ || /bin/true
```
Comment (omertuc) [inline Makefile:24]: `format` should be like in `assisted-service`:

```
format:
	golangci-lint run --fix -v
```

**Comment by omertuc** [comment]
/hold
Prevent auto-merge until my review is resolved

File: Makefile (hunk 1 above)
Comment (ronniel1) [reply Makefile:24]: nice.
Changed

**Comment by omertuc** [comment]
Not exactly what I meant, we still have separate `lint:` and `format:` in the service, but there's no real reason to have that separation that I know of, so no big deal.

**Review by YuviGold — APPROVED** [review]

**Pull request merged**